src/core/src/main/java/org/apache/jmeter/report/dashboard/HtmlTemplateExporter.java

```diff
@@ -145,33 +140,32 @@ public final void setExtraOptions(SubConfiguration extraOptions) {
          */
         @Override
         public ResultData customizeResult(ResultData result) {
+            if (extraOptions == null) {
```

Comment:
This is changing the behaviour. Previously it would add the `result` as `DATA_CTX_RESULT`.

Reply:
Good spot, now fixed.